app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+
+  def present(model, presenter_class=nil)
```

Comment:
Nadpisywanie metody `present` wydaje mi się odrobinę niebezpieczne. Wiem że taki pomysł łatwo przychodzi do głowy kiedy mówimy o `Presenter`-ach, ale łatwo można tą metodę pomylić z `present?` które sprawdza istnienie obiektu. Z takim językiem żyjemy i trzeba na to uważać ;)

Dodatkowo w innym komentarzu wyjaśnię czemu uważam że tej metody raczej w ogóle tu nie powinno być.